rpc/testing.go

```diff
@@ -26,7 +26,7 @@ import (
 
 // NewTestContext returns a rpc.Context for testing.
 func NewTestContext(t *testing.T) *Context {
-	tlsConfig, err := security.LoadTLSConfigFromDir(security.EmbeddedPrefix + "test_certs")
+	tlsConfig, err := security.LoadTLSConfigFromDir("test_certs")
```

Comment:
@MBerhault is refactoring the stuff anyways, I'll wait for the dust to settle. What's more worrying is that we currently **do** compile in the test certs everywhere, because it's used in the `rpc/testing` package. Will change that though.